SocialFilter/sections/css_extended.txt

```diff
@@ -3,6 +3,7 @@
 !
 video.wp.pl#?#article[data-testid="article-main"] > div[class^="sc-"] > div[class^="sc-"] > div[font-family="primary"] > div[class^="sc-"] > div[class^="sc-"] > div[class^="sc-"]:has(> button[shape="circle"] > svg[color="socialFacebook400"])
 w3schools.com#?##right > .sidesection:has(> .sharethis)
+aif.ua#?#.sharing_item:has(> a:not(.top_print_button))
```

Comment:
Will break href="http://www.aif.ru/contacts#anchor=error_form"

Reply:
better hide whole widget. Don't exclude parts of it